prototype/p2p/network_lifecycle.go

```diff
@@ -0,0 +1,61 @@
+package p2p
+
+import (
+	"pocket/p2p/types"
+	"strings"
+)
+
+func (g *P2PModule) configure(protocol, address, external string, peers []string) {
```

Comment:
The variable is name `g`, I'm guessing for `gater` but it is an element of the `P2PModule` struct.

1. Rename to either `m` or `module` or `p`. 
2. `P2PModule` struct should not be exposed outside the `p2p` module.